src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyShares.tsx

```diff
@@ -211,7 +216,7 @@ const KeyShareFlow = () => {
         setWarningMessage(warningTextMessage);
         setValidatorsCount(Object.values(validators).filter((validator: ValidatorType) => validator.isSelected).length);
         setSelectedOperatorsData(operatorsData.sort((operatorA: SelectedOperatorData, operatorB: SelectedOperatorData) => Number(operatorA.operatorId) - Number(operatorB.operatorId)));
-        setMaxAvailableValidatorsCount(maxValidatorsCount);
+        setMaxAvailableValidatorsCount(Object.values(validators).filter((validator: ValidatorType) => validator.isSelected).length < maxValidatorsCount ? Object.values(validators).filter((validator: ValidatorType) => validator.isSelected).length : maxValidatorsCount);
```

Comment:
can you make Object.values(validators).filter((validator: ValidatorType) => validator.isSelected).length a const? you are calculating it twice.

Reply:
+